.kitchen.yml

```diff
@@ -22,12 +22,14 @@ suites:
   - name: install
     run_list:
       - recipe[cerner_splunk_ingredient_test::splunk]
-      - recipe[cerner_splunk_ingredient_test::universal_forwarder]
   - name: uninstall
     run_list:
-      - recipe[cerner_splunk_ingredient_test::splunk]
```

Comment:
I need to look at this again... It isn't necessary because I updated the uninstall recipe to install before uninstall, but I think the main reason for doing that instead of keeping them separate was because the install recipe is set up so that it triggers a delayed restart (updating the ulimit) and that caused issues when uninstalling because it would then try to restart the splunk that didn't exist. I don't remember if I had a solution for that, or if I just decided "don't uninstall in the same recipe you're updating splunk".